### PR TITLE
Changelog 52

### PR DESCRIPTION
**Pull request by mfeifer**
First pass at changelog for this repository

[skip ci]

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+## Unreleased - as of Sprint 52 end 2017-01-14
+
+### Added
+- Inventory collection refresh using inventory abstraction ([#102](https://github.com/ManageIQ/manageiq-providers-azure/pull/Inventory collection refresh using inventory abstraction))
+- Inventory abstraction for data fetching and storing in inventory collections ([#98](https://github.com/ManageIQ/manageiq-providers-azure/pull/Inventory abstraction for data fetching and storing in inventory collections))
+- Map unknown event to automate to ConfigurationItemChangeNotification ([#93](https://github.com/ManageIQ/manageiq-providers-azure/pull/map event unknown to automate to ConfigurationItemChangeNotification))
+
+
+### Changed
+- Rename dto to inventory ([#95](https://github.com/ManageIQ/manageiq-providers-azure/pull/Rename dto to inventory))
```
Comment (blomquisg) [inline CHANGELOG.md]: "Rename 'DTO' to 'Inventory' in refresh"

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+## Unreleased - as of Sprint 52 end 2017-01-14
+
+### Added
+- Inventory collection refresh using inventory abstraction ([#102](https://github.com/ManageIQ/manageiq-providers-azure/pull/Inventory collection refresh using inventory abstraction))
```
Comment (blomquisg) [inline CHANGELOG.md]: Wow that's a confusing PR title.  How about:

"Collect inventory using 'inventory' abstraction in refresh"

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+- Inventory abstraction for data fetching and storing in inventory collections ([#98](https://github.com/ManageIQ/manageiq-providers-azure/pull/Inventory abstraction for data fetching and storing in inventory collections))
```
Comment (blomquisg) [inline CHANGELOG.md]: "Introduce 'inventory' abstraction for fetching and storing inventory data"

File: CHANGELOG.md (hunk 1 above)
Comment (mfeifer) [inline CHANGELOG.md]: 👍

File: CHANGELOG.md (hunk 2 above)
Comment (mfeifer) [inline CHANGELOG.md]: Thanks for the translation!
👍

File: CHANGELOG.md (hunk 3 above)
Comment (mfeifer) [inline CHANGELOG.md]: 👍

**Comment by bronaghs** [comment]
@blomquisg @mfeifer - looks like the PR links are pointing to the azure repo instead of the amazon repo

**Comment by mfeifer** [comment]
Thanks @bronaghs . So many changelogs, so little time....I'll make a new PR with the updated links

**Comment by mfeifer** [comment]
@miq-bot add_label documentation

**Pull request merged**